DatadogInternal/Sources/NetworkInstrumentation/URLSession/DatadogURLSessionDelegate.swift

```diff
@@ -39,17 +46,8 @@ open class DatadogURLSessionDelegate: NSObject, URLSessionDataDelegate {
     @objc
     override public init() {
         core = nil
-        firstPartyHosts = .init()
-
-        URLSessionInstrumentation.enable(
-            with: .init(
-                delegateClass: Self.self,
-                firstPartyHostsTracing: .traceWithHeaders(hostsWithHeaders: firstPartyHosts.hostsWithTracingHeaderTypes)
-            ),
-            in: core ?? CoreRegistry.default
-        )
-
         super.init()
+        try? swizzle(firstPartyHosts: .init())
```

Comment:
blocking/ At minimum we should log the problem to user console. This could save hours in troubleshooting user issues.

Reply:
Indeed 👍